README.md

```diff
@@ -80,8 +81,8 @@ pip install guardrails-ai
     First, install the necessary guardrails from Guardrails Hub.
 
     ```bash
-    guardrails hub install hub://guardrails/competitor_check
-    guardrails hub install hub://guardrails/toxic_language
+    guardrails hub install hub://guardrails/regex_match
```

Comment:
@thekaranacharya what's the reason for the switch to `regex_match` and `valid_length`?

Reply:
@ShreyaR looks like in the current README we show installing `competitor_check` and `toxic_language` but then below that show using `regex_match` and `valid_length`.  This change synchronizes the install with the usage

Reply:
Ah great catch! In that case can we update the example usage so that we use the CompetitorCheck and ToxicLanguage validators?

The reason for biasing towards these validators is that they're good starter examples for showing useful but technically complex validators. We've received feedback in the past that the structural validators undermine how complex validators can generally be.